lib/internal/cluster/master.js

```diff
@@ -12,6 +12,18 @@ const cluster = new EventEmitter();
 const intercom = new EventEmitter();
 const SCHED_NONE = 1;
 const SCHED_RR = 2;
+var debugPortOffset = 0;
+
+const debugArgRegex = /^(--inspect|--debug|--debug-(?:brk|port))/;
```

Comment:
I think I could chahge it to `/^(--inspect|--debug)/` and it would work.

If you're suggesting using strict form, how about this one? /^(--inspect|--debug|--debug-(?:brk|port))(?:=.*)?$/